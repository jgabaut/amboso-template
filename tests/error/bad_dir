#!/bin/bash
"./anvil" -D "wrongdirectory" -q 0.1.0
