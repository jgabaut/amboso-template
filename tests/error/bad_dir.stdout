^[[1;31m[ERROR]    Can't find "stego.lock" in ( wrongdirectory ).^[[0m$
$
